src/init/population.rs

```diff
@@ -179,6 +184,23 @@ fn parse_u64_or_na<'de, D: Deserializer<'de>>(d: D) -> Result<u64, D::Error> {
     )))
 }
 
+/// Parses either a float or the string "NA". "NA" maps to 0. It's fine if 0 also appears as a
```

Comment:
Originally this was like the method above, failing if it sees 0 as a value. But I found that happening for the case I reported in Slack, so for now, allowing this